cmd/jaeger/internal/integration/query_test.go

```diff
@@ -25,22 +25,17 @@ func TestJaegerQueryService(t *testing.T) {
 	t.Log("Query initialized")
 
 	// Start another instance of Jaeger receiving traces and write traces to Remote Storage
```

Comment:
can you do the diagrams in the README for how query and grpc test setups are done?

Reply:
@yurishkuro I booked https://github.com/jaegertracing/jaeger/issues/7055 for my self to do so in a follow-up PR